lib/libutil/kinfo_getproc.3

```diff
@@ -42,15 +42,177 @@ This function is used for obtaining process information from the kernel.
+	int	ki_structsize;		/* size of this structure */
+	int	ki_layout;		/* reserved: layout identifier */
+	pid_t	ki_pid;			/* process identifier */
+	pid_t	ki_ppid;		/* parent process identifier */
+	pid_t	ki_pgid;		/* process group identifier */
+	pid_t	ki_tpgid;		/* tty process group identifier */
+	pid_t	ki_sid;			/* session identifier */
+	pid_t	ki_tsid;		/* terminal session identifier */
+	sigset_t ki_siglist;		/* signals arrived but not delivered */
+	sigset_t ki_sigmask;		/* current signal mask */
+	sigset_t ki_sigignore;		/* signals being ignored */
+	sigset_t ki_sigcatch;		/* signals being caught by user */
+	uid_t	ki_uid;			/* effective user id */
+	uid_t	ki_ruid;		/* real user id */
+	uid_t	ki_svuid;		/* saved effective user id */
+	gid_t	ki_rgid;		/* real group id */
+	gid_t	ki_svgid;		/* saved effective group id */
+	short	ki_ngroups;		/* number of groups */
+	gid_t	ki_groups[KI_NGROUPS];	/* groups */
+	vm_size_t ki_size;		/* virtual size */
+	segsz_t	ki_rssize;		/* resident set size in pages */
+	segsz_t	ki_swrss;		/* resident set size before last swap */
+	segsz_t	ki_tsize;		/* text size (pages) */
+	segsz_t	ki_dsize;		/* data size (pages) */
+	segsz_t	ki_ssize;		/* stack size (pages) */
+	u_short	ki_xstat;		/* exit status for wait & stop signal */
+	u_short	ki_acflag;		/* accounting flags */
+	fixpt_t	ki_pctcpu;		/* %cpu for process during ki_swtime */
+	u_int64_t ki_runtime;		/* real time in microsec */
+	struct	timeval ki_start;	/* starting time */
+	struct	timeval ki_childtime;	/* time used by process children */
+	long	ki_flag;		/* P_* flags */
+	long	ki_kiflag;		/* KI_* flags */
+	int	ki_traceflag;		/* kernel trace points */
+	char	ki_stat;		/* S* process status */
+	signed char ki_nice;		/* process "nice" value */
+	char	ki_lock;		/* process lock (prevent swap) count */
+	char	ki_tdname[TDNAMLEN+1];	/* thread name */
+	char	ki_wmesg[WMESGLEN+1];	/* wchan message */
+	char	ki_login[LOGNAMELEN+1];	/* setlogin name */
+	char	ki_lockname[LOCKNAMELEN+1]; /* lock name */
+	char	ki_comm[COMMLEN+1];	/* command name */
+	char	ki_emul[KI_EMULNAMELEN+1]; /* emulation name */
+	char	ki_loginclass[LOGINCLASSLEN+1]; /* login class */
+	uint64_t ki_tdev;		/* controlling tty dev */
+	int	ki_oncpu;		/* which cpu we are on (machdep) */
+	int	ki_lastcpu;		/* last cpu we were on (machdep) */
+	int	ki_tracer;		/* pid of tracing process */
+	int	ki_flag2;		/* P2_* flags */
+	int	ki_fibnum;		/* default FIB number */
+	int	ki_jid;			/* process jail ID */
+	int	ki_numthreads;		/* number of threads in total */
+	lwpid_t	ki_tid;			/* thread id */
+	struct	priority ki_pri;	/* process priority */
+	struct	rusage ki_rusage;	/* process rusage statistics */
+	struct	rusage ki_rusage_ch;	/* rusage of children processes */
+	long	ki_sflag;		/* PS_* flags */
+	/* ... additional fields, see <sys/user.h> for full definition.
+	 * Use ki_structsize or KINFO_PROC_SIZE for actual struct size. */
+};
+.Ed
+.Pp
+Key fields include:
+.Bl -tag -width ".Va ki_rusage"
```

Comment:
```suggestion
.Bl -tag -width "ki_rusage"
```
Macros in width fields is an undefined behavior. I'm trying to get rid of all the doc that erroneously says to do that. Where did you learn to do that?